tests/e2e/e2e_test.go

```diff
@@ -101,6 +118,16 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 		ginkgo.DeferCleanup(func() {
 			tests.Outf("Shutting down network\n")
 			require.NoError(network.Stop())
+			if archiveNetworkDirOnTeardown {
+				archivePath, err := filepath.Abs("../../testnet.tar.xz")
```

Comment:
Hmmmm I feel like using local/parent file paths is pretty gross... because of the added assumptions on where the binary is executed